src/api/bkuser_core/categories/plugins/local/syncer.py

```diff
@@ -235,8 +235,9 @@ def _sync_users(self, parser_set: "ParserSet", users: list):
             progress(index, total, f"loading {username}")
             try:
                 updating_profile = Profile.objects.get(username=username, category_id=self.category_id)
-
-                # 如果已经存在，则更新该 profile
+                # 已存在的用户：如果未勾选 <进行覆盖更新>（即is_overwrite为false）=》则忽略，反之则更新该 profile
+                if not is_overwrite:
```

Comment:
logger.debug("username %s exist, and is_overwrite is false, so will not do update for this user, skip", username)

Reply:
Done